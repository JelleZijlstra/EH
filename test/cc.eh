#!/usr/bin/ehi
# This program previously ran into a GC bug that caused ehi to crash with an assertion failure.

echo (GC = GarbageCollector)
echo(GC.run())
echo(CountClass.new())
echo(GC.run())
#echo GC.run()
#echo GC.run()
CountClass.new()
GC.run()
String.length()
