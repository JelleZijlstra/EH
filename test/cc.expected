0x800c20

0x800cb0

Uncaught exception of type TypeError: Invalid base object for "String.length": Null
